config/initializers/floe_docker_runner.rb

```diff
@@ -0,0 +1,21 @@
+def floe_docker_runner
```

Comment:
So "docker" in this case is talking about the resource that is being run so a docker image specified by "docker://..." in the ASL file, not about what ends up being run.  A little bit of name overloading unfortunately.

Reply:
I think the same still applies and would argue that `container://...` seems more appropriate.

Reply:
If we did decide to change the resource scheme name for a docker image I'm not sure I'd start with this method.  We'd need to start with what we use in the ASL files first which would live in the floe gem so that'd be a place to start the discussion.

Podman does use `docker://` as the scheme for a remote image URI which is what we're using so this is at least a cross-vendor recognized format [[ref]](https://docs.podman.io/en/latest/markdown/podman-run.1.html)

> docker://docker-reference (Default) An image reference stored in a remote container image registry.

Reply:
While I'd agree container:// is probably clearer, it is common practice to use docker:// (podman example above and also skopeo uses that format), so I agree with keeping docker://